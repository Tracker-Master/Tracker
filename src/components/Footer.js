import React from 'react';
// import { Link } from 'react-router-dom';

// import { StyledFooter } from 'Styles/components/FooterStyles';
import { StyledFooter } from '../styles/components/FooterStyles';

const Footer = () => {
  return (
    <StyledFooter>
      <small>
        Your Trello board needs only three lists: <br />
        <em>
          <strong>To Do</strong>, <strong>Doing</strong>,{' '}
          <strong>Done</strong>
        </em>
      </small>
    </StyledFooter>
  );
};

export default Footer;
