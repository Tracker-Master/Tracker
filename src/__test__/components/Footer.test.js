import React from 'react';
import { mount, shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Footer from '../../components/Footer';

describe('<Footer />', () => {
  const footer = shallow(<Footer />);

  test('Footer rendering', () => {
    expect(footer.length).toBe(1);
  });

  const tree = renderer.create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot();

});