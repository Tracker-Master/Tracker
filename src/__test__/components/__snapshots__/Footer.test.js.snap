// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  line-height: 0.9rem;
}

.c0 small {
  color: #757575;
}

@media (min-width:768px) {

}

@media (min-width:1024px) {
  .c0 {
    background: #673AB7;
  }

  .c0 a small {
    color: #fff;
  }
}

<div
  className="c0"
>
  <small>
    Your Trello board needs only three lists: 
    <br />
    <em>
      <strong>
        To Do
      </strong>
      , 
      <strong>
        Doing
      </strong>
      ,
       
      <strong>
        Done
      </strong>
    </em>
  </small>
</div>
`;
